curriculum/challenges/english/25-front-end-development/lab-house-painting/66d6a7a3e1aa411e94bf2346.md

```diff
@@ -0,0 +1,469 @@
+---
+id: 66d6a7a3e1aa411e94bf2346
+title: Build a House Painting
+challengeType: 14
+dashedName: build-a-house-painting
+demoType: onClick
+---
+
+# --description--
+
```

Comment:
A few lines about the project would be good to add.
I came up with this, feel free to adjust it.

```
In this lab, you will use HTML and CSS positioning to create a house. There would be a parent element that would contain the individual elements of the house like windows and doors. You will apply styles like `position`, `width`, `height`, `border`, and `background-color` to the parent element and its children to create the house.
```